AutoInstaller.ps1

```diff
@@ -0,0 +1,38 @@
+#Requires -RunAsAdministrator
+
+# Check user has webhook URL ready
+$userPrompt = Read-Host -Prompt "Do you have your webhook URL ready? Y/N"
+if ($userprompt -ne 'Y') {
+    Write-Output "Please create a Discord webhook before continuing. `nFull instructions avalible at https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks"
+    exit
+}
+# Get latest release from GitHub
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+$latestRelease = Invoke-WebRequest -Uri https://github.com/tigattack/VeeamDiscordNotifications/releases/latest -Headers @{"Accept"="application/json"} -UseBasicParsing
+# Release IDs are returned in a format of {"id":3622206,"tag_name":"v1.0"} so we need to extract tag_name.
+$latestVersion = $latestRelease.Content | ConvertFrom-Json | Select-Object tag_name -ExpandProperty tag_name
+# Pull latest version of script from GitHub
+Invoke-WebRequest -Uri https://github.com/tigattack/VeeamDiscordNotifications/releases/download/$LatestVersion/VeeamDiscordNotifications-$LatestVersion.zip -OutFile $PSScriptRoot\VeeamDiscordNotifications-$LatestVersion.zip
+
+# Expand downloaded ZIP and cleanup
+Expand-Archive $PSScriptRoot\VeeamDiscordNotifications-$LatestVersion.zip -DestinationPath C:\VeeamScripts
+Rename-Item C:\VeeamScripts\VeeamDiscordNotifications-$LatestVersion C:\VeeamScripts\VeeamDiscordNotifications
+Remove-Item $PSScriptRoot\VeeamDiscordNotifications-$LatestVersion.zip
+# Assign webhook url to variable
+$webhookurl = Read-Host -Prompt "Please paste your Webhook URL now"
+
+# Get the config file and write the user webhook
+$Config = (Get-Content C:\VeeamScripts\VeeamDiscordNotifications\config\conf.json) | ConvertFrom-Json

```

Comment:
```suggestion
$Config = Get-Content "C:\VeeamScripts\VeeamDiscordNotifications\config\conf.json" -Raw | ConvertFrom-Json
```

* Using `-Raw` preserves the formatting of the source file. Not strictly necessary but generally it's safer to use this flag in such cases.
* Best to use quotes around file paths. Again, not strictly necessary in this specific case, but it's a general good practice.
* `Get-Content blahblah` doesn't need to be in brackets.